src/components/organisms/SettingsChange.tsx

```diff
@@ -0,0 +1,303 @@
+import React, { useState, useEffect } from "react";
+import styles from "@/styles/components/organisms/Settings.module.scss";
+import { styled } from "@stitches/react";
+import { indigo, mauve, blackA } from "@radix-ui/colors";
+import * as TabsPrimitive from "@radix-ui/react-tabs";
+import * as CheckboxPrimitive from "@radix-ui/react-checkbox";
+import { CheckIcon } from "@radix-ui/react-icons";
+import api from "../../lib/axios_settings";
```

Comment:
Please change `"../lib/axios_settings"` to `"@/lib/axios_settings"`

Reply:
Done